openslides/core/config_variables.py

```diff
@@ -126,7 +124,7 @@ def get_config_variables():
             {'value': 'left', 'display_name': 'Left'},
             {'value': 'center', 'display_name': 'Center'},
             {'value': 'right', 'display_name': 'Right'}),
-        weight=146,
+        weight=145,
```

Comment:
Can we please change the values so that there is some space between them again. The intention is that plugins may go inbetween with their values.